README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+# News Expolrer Frontend
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Overview
 
-## Available Scripts
+This is the frontend part of the News Explorer project. This is a platform that allows you to search for the latest news articles from an external API call to the server. With a simple and easy-to-use interface, you can quickly find the information you need on a variety of topics with the ability to save them for later and if the user no longer interested in some article they have the option to delete it as well.
+Project implemented mainly in React with a use of Context API.
 
-In the project directory, you can run:
+## Technologies
 
-### `npm start`
+- HTML
+- CSS
+- JavaScript
+- React
+- Context API
+- Hooks
+- React Router
+- Webpack
+- Babel
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Features
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- Search for articles by keyword or topic
+- Register and Login to personal account
+- Save articles for later
+- Delete no longer needed articles
+- Frontend fetches data from a RESTFull News Explorer API
+- Dynamic content and componets rendering
+- Responsive design
+- Form validation
+- Error handling
+- Open links in new tab
 
-### `npm test`
+## Live Demo
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Fully functional Fullstack app (GCP):[https://amitaynews.students.nomoredomainssbs.ru/](https://amitaynews.students.nomoredomainssbs.ru/)
```

Comment:
The link to your deployed app leads to a `forbidden page`. See here https://disk.yandex.ru/i/PTLUnhsGTjxGbw